src/components/TripList.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import TripListItem from './TripListItem'
+import * as Trips from '../data/trips.json'
```

Comment:
I think this can be simply imported as `import Trips from '../data/trips.json'`. Maybe rename the imported data to TripsData (see comment below)